### ISSUE TITLE
jscodeshift-imports extension publishes uncompiled ES6 code

### ISSUE THREAD

**Issue by rylanc** [cpojer/js-codemod#24]
When trying to use jscodeshift-imports with my own transforms, node chokes on unimplemented ES6 features:
```
node_modules/jscodeshift-imports/addImport.js:3
const {statements, statement} = j.template;
      ^

SyntaxError: Unexpected token {
```

**Comment by cpojer** [comment]
cc @pieterv 

jscodeshift runs codemods through babel, so I'm not sure why you are seeing this problem.

**Comment by pieterv** [comment]
Maybe since it's a npm module it doesn't transform it. I guess we could remove the deconstruction.

**Comment by fkling** [comment]
@pieterv: yeah, I think if you use babel register, it ignores files in `node_modules`. Maybe this can be configured, don't know. Alternatively, use only ES6 features that are supported in Node or publish the compiled version.

**Comment by pieterv** [comment]
@fkling yeah i think it seems reasonable to just use the features that are supported by latest node.

**Comment by Daniel15** [comment]
> Maybe since it's a npm module it doesn't transform it

The default Babel config ignores node_modules, see the comments at https://babeljs.io/docs/usage/require/. It would be super slow if it had to process all third-party modules, especially given the module duplication issues in npm v2.

In this case we could just stop relying on syntax sugar :stuck_out_tongue:

**Comment by pieterv** [comment]
Hey @rylanc, #29 should fix this. I just published it as `v1.0.1` let me know if you have any problems with it.
